File: app/src/main/java/com/battlelancer/seriesguide/shows/overview/SeasonTools.java
package com.battlelancer.seriesguide.shows.overview;

import android.content.Context;
import com.battlelancer.seriesguide.R;

public class SeasonTools {

    public static boolean hasSkippedTag(String tags) {
        return SeasonTags.SKIPPED.equals(tags);
    }

    /**
     * Builds a localized string like "Season 5" or if the number is 0 "Special Episodes".
     */
    public static String getSeasonString(Context context, int seasonNumber) {
        if (seasonNumber == 0) {
            return context.getString(R.string.specialseason);
        } else {
            return context.getString(R.string.season_number, seasonNumber);
        }
    }

}

File: app/src/main/java/com/battlelancer/seriesguide/shows/overview/SeasonTools.kt
// SPDX-License-Identifier: Apache-2.0
// Copyright 2013, 2014, 2016, 2018, 2022-2024 Uwe Trottmann

package com.battlelancer.seriesguide.shows.overview

import android.content.Context
import com.battlelancer.seriesguide.R

object SeasonTools {

    /**
     * Builds a localized string like "Season 5" or if the number is 0 "Special Episodes".
     */
    fun getSeasonString(context: Context, seasonNumber: Int): String {
        return if (seasonNumber == 0) {
            context.getString(R.string.specialseason)
        } else {
            context.getString(R.string.season_number, seasonNumber)
        }
    }

}